src/lib/zAuction.ts

```diff
@@ -89,10 +69,14 @@ function getNftId(contract: string, tokenId: string) {
 	return nftId;
 }
 
-export async function getBidsForNft(contract: string, tokenId: string) {
+export async function getBidsForNft(
+	contract: string,
+	tokenId: string,
+	apiName: string,
```

Comment:
Please rename this to `baseApiUri` instead of `apiName`.

Also; Please move the `baseApiUri` parameter to be the first parameter in all of these function calls.